### PR TITLE
Adds documentation regarding staging project creation and copy from template

### PR DESCRIPTION
**Pull request by saraycp**
Adds documentation about staging project creation and copy from template.

It also fix a typo and optimize images.

### CODE REVIEW

File: xml/obs_staging_workflow.xml [hunk 1]
```diff
@@ -57,7 +57,7 @@ geeko > <command>osc</command> api '/staging/openSUSE:Factory:Staging/staging_pr
   </sect2>
 
   <sect2 xml:id="sec.obs.staging.workflow.projects.show">
-    <title>Overview of All Staging Projects</title>
+    <title>Overview of a Single Staging Projects</title>
```
Comment (eduardoj) [inline xml/obs_staging_workflow.xml]: [nitpick] Typo: s/Projects/Project/

File: xml/obs_best_practice_staging_workflow.xml [hunk 2]
```diff
@@ -202,5 +202,43 @@
...
       </listitem>
     </itemizedlist>
+    <sect3>
+      <title>Create Staging Project from Scratch</title>
+      <para>
+        Right after the creation of a Staging, two new staging projects are
+        automatically created and assigned to it: Staging:A and Staging:B. However,
+        it is also possible to create a new staging project from scratch.
+
+        Being placed on the Staging dashboard, click on
```
Comment (eduardoj) [inline xml/obs_best_practice_staging_workflow.xml]: [nitpick] Add an ending para tag and and starting para tags?

`</para><para>`

Comment (saraycp) [reply xml/obs_best_practice_staging_workflow.xml]: sure!

File: xml/obs_best_practice_staging_workflow.xml [hunk 3]
```diff
@@ -202,5 +202,44 @@
...
     </itemizedlist>
+    <sect3>
+      <title>Create Staging Project from Scratch</title>
+      <para>
+        Right after the creation of a Staging, two new staging projects are
+        automatically created and assigned to it: Staging:A and Staging:B. However,
+        it is also possible to create a new staging project from scratch.
+      </para>
+      <para>
+        Being placed on the Staging dashboard, click on
```
Comment (dmarcoux) [inline xml/obs_best_practice_staging_workflow.xml]: Do you mean "on the Staging dashboard"? "Being placed on" sounds weird.

**Pull request merged**